Source/WebCore/platform/graphics/mac/controls/ControlMac.mm

```diff
@@ -231,14 +231,18 @@ static void drawCellOrFocusRing(GraphicsContext& context, const FloatRect& rect,
         return;
     }
 
-    auto imageBufferRect = FloatRect { { 0, 0 }, rect.size() };
+    static constexpr float focusRingOutlineWidth = 3;
```

Comment:
Unfortunate that this needs to be hardcoded, but this matches the `ThemeMac` implementation, and I'm unaware of a way to get this from the system.